test/hotspot/jtreg/compiler/vectorization/runner/ArrayCopyTest.java

```diff
@@ -73,6 +75,8 @@ public ArrayCopyTest() {
 
     // ---------------- Simple Copy ----------------
     @Test
+    @IR(applyIfCPUFeatureOr = {"asimd", "true", "avx2", "true"},
```

Comment:
For X86 you can replace avx2 with sse for store vector checks.

Reply:
This is also updated in my last commit. Could you help verify these again?